src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -25,6 +25,8 @@ public class TemplateWriter
     {
         public const string GeneratorMetadataPath = "metadata._generator";
         public const string NestedDeploymentResourceType = AzResourceTypeProvider.ResourceTypeDeployments;
+        
```

Comment:
should we add a test to make sure no one accidentally changes this?

Reply:
I thought about it, but the tests that do that sort of 1:1 assertion don't add much value in general. Also all the module baseline tests will fail because emitted JSON will be different.

Reply:
>all the module baseline tests will fail because emitted JSON will be different.

Good point